src/TimeGrid.jsx

```diff
@@ -77,9 +80,23 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
+    }
+
     //this.checkOverflow()
   }
 
+  componentWillReceiveProps(nextProps) {
+    // When paginating, reset scroll
+    if (+nextProps.start !== +this.props.start) {
```

Comment:
hm... what I want to prevent is an unexpected scroll change when the user has already scrolled down somewhere else, so if you think it's important or that `scrollTime` can change in between renders, I can scroll to the new position if the user hasn't scrolled at all. Although sounds to me like a lot of unneeded complexity.